tests/phpunit/integration/Core/Tracking/REST_Tracking_Consent_ControllerTest.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+/**
+ * REST_Tracking_Consent_ControllerTest
+ *
+ * @package   Google\Site_Kit\Tests\Core\Tracking
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\Tracking;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\REST_API\REST_Route;
+use Google\Site_Kit\Core\REST_API\REST_Routes;
+use Google\Site_Kit\Core\Tracking\REST_Tracking_Consent_Controller;
+use Google\Site_Kit\Core\Tracking\Tracking_Consent;
+use Google\Site_Kit\Tests\Fake_Site_Connection_Trait;
+use Google\Site_Kit\Tests\TestCase;
+use WP_REST_Request;
+use WP_REST_Server;
+
+/**
+ * @group tracking
+ */
+class REST_Tracking_Consent_ControllerTest extends TestCase {
+	use Fake_Site_Connection_Trait;
+
+	public function tearDown() {
+		parent::tearDown();
+		// This ensures the REST server is initialized fresh for each test using it.
+		unset( $GLOBALS['wp_rest_server'] );
+		unset( $GLOBALS['current_user'] );
+	}
+
+	public function test_register() {
+		$tracking_consent_mock = $this->getTrackingConsentMock( array( 'register', 'get' ) );
+		$tracking_consent_mock->expects( $this->once() )->method( 'register' );
+		$REST_Controller = new REST_Tracking_Consent_Controller( new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
+		$this->force_set_property( $REST_Controller, 'consent', $tracking_consent_mock );
+
+		remove_all_filters( 'googlesitekit_apifetch_preload_paths' );
+		remove_all_filters( 'googlesitekit_rest_routes' );
+
+		$REST_Controller->register();
+
+		$rest_routes = apply_filters( 'googlesitekit_rest_routes', array() );
+
+		$routes = array_map(
+			function ( REST_Route $route ) {
+				return $route->get_uri();
+			},
+			$rest_routes
+		);
+		$this->assertEqualSets(
+			array(
+				'core/user/data/tracking',
+			),
+			$routes
+		);
+		$this->assertTrue( has_filter( 'googlesitekit_apifetch_preload_paths' ) );
+		$this->assertTrue( has_filter( 'googlesitekit_rest_routes' ) );
+	}
+
+	public function test_unauthorized_get_request() {
+		$REST_Controller = new REST_Tracking_Consent_Controller( new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
```

Comment:
Let's use lower case variables: 
```suggestion
		$controller = new REST_Tracking_Consent_Controller( new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
```

Same applies to all occurrences in this file.